packages/manager/modules/bm-server-components/src/service-status/service-status.html

```diff
@@ -54,12 +54,13 @@
 
     <!--Monitoring options-->
     <oui-tile-definition
+        class="overide-flex"
```

Comment:
Can you use a scoped class? Define a class for the page and define the required css class inside the page's class.